presets.json

```diff
@@ -25,9 +25,9 @@
     "accent": "#6F49AB"
   },
   "Teal:" {
```

Comment:
The bug is still there; Colon is inside the key, should be outside of it. This causes JSON parsing error ;-)